kong/tools/utils.lua

```diff
@@ -136,6 +136,42 @@ do
   end
 end
 
+
+function _M.get_system_trusted_certs_filepath()
+  local ok, code, stdout, stderr =
+    pl_utils.executeex("openssl version -d")
+  if not ok or code ~= 0 or type(stdout) ~= "string" then
+    return nil, stderr
+  end
+
+  local folder = stdout:match('OPENSSLDIR: "(.+)"')
```

Comment:
I was loitering and saw this PR 😄 .
 `openssl version -d` returns the hardcoded marco and it may vary during configure, if user links Kong to a custom built openssl this will become unreliable. In fact, the CA bundle are usually packed by the distro and is separate from openssl itself. So I'd suggest to just use a hardcoded full path instead, without considering prefix.
References are https://golang.org/src/crypto/x509/root_linux.go and https://github.com/curl/curl/blob/7a71965e9761f80aa4508472c5b1e3ab03ba018a/acinclude.m4#L2151

Reply:
Your "loitering" is way more productive than mine! Thanks for the review, I have made the changes you suggested.